src/ctapipe/utils/tests/test_astro.py

```diff
@@ -3,9 +3,28 @@
 This module contains the utils.astro unit tests
 """
 from astropy import units as u
-from astropy.coordinates import SkyCoord
+from astropy.coordinates import Angle, SkyCoord
 
-from ..astro import get_bright_stars
+from ..astro import get_bright_stars, get_bright_stars_with_motion
+
+
+def test_get_bright_stars_with_motion():
+    """
```

Comment:
If you add more catalogs, it makes sense to test all of them. Also, it would be nice to test a but more details, or add more comments about the expected outcome of the query.

Reply:
oka, i will get to that part tomorrow morning